src/api/services/comments.ts

```diff
@@ -0,0 +1,42 @@
+import { IncidentCommander } from "../axios";
+import { resolve } from "../resolve";
+import { User } from "./users";
+
+interface NewComment {
+  user: User;
+  incidentId: string;
+  hypothesisId: string;
+  comment: string;
+}
+
+export interface Comment {
+  id: string;
+  created_by: User;
+  incident_id: string;
+  hypothesis_id: string;
+  comment: string;
+  created_at: string;
+  updated_at: string;
+}
+
+export const getCommentsByHypothesis = (hypothesisId: string) =>
+  resolve<Comment[]>(
+    IncidentCommander.get(
+      `/comment?select=*,created_by(id,name,avatar)&hypothesis_id=eq.${hypothesisId}&order=created_at.desc`
+    )
+  );
+
+export const createComment = async ({
```

Comment:
can we have type info here.

Reply:
```js
}: NewComment) =>
  resolve<Comment>(
```
those are the types. Did you mean something else?